engine/commands/chat_completion_cmd.h

```diff
@@ -9,8 +9,8 @@ class ChatCompletionCmd {
  public:
   void Exec(const std::string& host, int port, const std::string& model_handle,
             std::string msg);
-  void Exec(const std::string& host, int port, const config::ModelConfig& mc,
-            std::string msg);
+  void Exec(const std::string& host, int port, const std::string& model_handle,
+            const config::ModelConfig& mc, std::string msg);
```

Comment:
should be `const std::string& msg` ?

Reply:
we use std::move, so it is not a problem. But I think should use `const std::string&` like other parameters. Will fix it shortly

Reply:
ah no, it is intended to check the empty condition here
```
std::string user_input = std::move(msg);
      if (user_input.empty()) {
```